### DISCUSSION TITLE
no movement detcted

### DISCUSSION THREAD

**Discussion by githubnoobjk** [dcmartin/motion-ai#28] [Q&A]
solved:
Don't know what has changed but motion is detected and recognized from my first camera. Now trying to connect my other cameras. Not much succes so far.

I think I installed everything correct. However there is no detection of movement or something like this. Should I install something else besides motion-ai classic ?

Sorry for asking so much, but I think it is a great add-on when i succeed in installing it.

**Comment by dcmartin** [comment]
Are you running the AI's (yolo4motion, face4motion, alpr4motion) containers?

On the administrators panel item the AI tab should look like the following:

<img width="1433" alt="image" src="https://user-images.githubusercontent.com/427535/137056076-c5fb67a5-5b21-4fa3-bc31-f7b7b7032e9f.png">

**Reply by githubnoobjk** [reply]
I think it looks the same.
![image](https://user-images.githubusercontent.com/53235543/137197896-15b34e6f-c502-4618-b77a-0f41309d6de8.png)

**Reply by githubnoobjk** [reply]
Movement was detected now. Perhaps I was to impatient ?
